tools/cbmc/proofs/Queue/QueueGenericCreate/QueueGenericCreate_harness.c

```diff
@@ -0,0 +1,19 @@
+#include "FreeRTOS.h"
+#include "queue.h"
+#include "queue_datastructure.h"
+#include "cbmc.h"
+
+
+void harness(){
+	UBaseType_t uxQueueLength;
+	UBaseType_t uxItemSize;
+	uint8_t ucQueueType;
+	__CPROVER_assume(uxQueueLength > ( UBaseType_t ) 0);
+
+	/* The implicit assumption for the QueueGenericCreate method is,
+	   that there are no overflows in the computation and the inputs are safe.
+	   There is no check for this in the code base */
+	UBaseType_t upper_bound = portMAX_DELAY - sizeof(Queue_t);
```

Comment:
Why this upper bound? Is this for performance reasons, or required by the spec?

Reply:
portMAX_DELAY is the maximum value a UBaseType can get for a specific system. So this bound is resulting from the specs.

Reply:
That's a really odd name for the constant.  Is it possible to make a constant with a better name?